app/views/searches/show.html.haml

```diff
@@ -1,98 +1,90 @@
-!!!5
-- %i( reset site header icon list table ).each do |file|
-  = javascript_include_tag "https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/components/#{file}.min.js"
-  = stylesheet_link_tag "https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.1.8/components/#{file}.min.css"
-
-= stylesheet_link_tag 'http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css'
-= javascript_include_tag 'http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js'
-
-= javascript_include_tag 'https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'
-
-.ui.container
-  %h1.ui.header.page-header
-    = @search.title
-
-  #map{style: 'height: 360px; width: 750px;'}
-
-  %table.ui.celled.table.unstackable
-    %thead
+%h1.ui.header.page-header
+  = @search.title
+
+#map{style: 'height: 360px; width: 750px;'}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace